cmd/onos-config/onos-config.go

```diff
@@ -145,6 +156,27 @@ func main() {
 	}
 }
 
+func newDialOptions(clientCrtPath, clientKeyPath *string) ([]grpc.DialOption, error) {
```

Comment:
I think you can reuse **HandleCertArgs()** from pkg/certs/clientcerts.go instead. And it has the advantage of loading a default client1.crt and client1.key internally if no argument is given